src/lib.rs

```diff
@@ -538,3 +538,68 @@ pub fn fuzzer_mutate(data: &mut [u8], size: usize, max_size: usize) -> usize {
     assert!(new_size <= data.len());
     new_size
 }
+
+/// Define a custom cross-over function to combine test cases.
+///
+/// This is optional, and libFuzzer will use its own, default cross-over strategy
+/// if this is not provided. (As of the time of writing, this default strategy
+/// takes alternating bytes sequences from the two test cases, to construct the
+/// new one)
+///
+/// This could potentially be useful if your input is, for instance, a
+/// sequences of fixed sized, multi-byte values and the crossover could then
+/// merge a discrete number of values rather than joining parts of a value.
+#[macro_export]
+macro_rules! fuzz_crossover {
```

Comment:
Can you add a simple example, similar to the docs for `fuzz_mutator!`?

Reply:
Apologies for the delay, but I have added an example